stdlib/http/src/test/resources/test-src/services/nativeimpl/cookie/cookie-native-function-negative.bal

```diff
@@ -248,7 +248,6 @@ function testAddPersistentCookieWithoutPersistentStore() returns @tainted http:C
     }
     return cookies;
 }
-
 //TODO Table remove - Fix
 //function testRemovePersistentCookieFromCookieStore_1() returns @tainted http:Cookie[] {
```

Comment:
Why this test case is commented out?

Reply:
This test case checked the error message when removing a non-existing cookie. However, with the new table, it panics if we try to remove an entry from the table if the key does not exist hence the test case failed. Updated in 9a986a0 to assert the runtime exception.